### ISSUE TITLE
Tutorial - Knight is being dragged with background

### ISSUE THREAD

**Issue by 3vilguy** [react-dnd/react-dnd#1129]
**Describe the bug**
I'm following the tutorial and everything is fine until the point when we make knight draggable for the first time. According to tutorial only knight symbol should be draggable, but in my local version you can clearly see white background behind the symbol.

**To Reproduce**
1. Follow the [tutorial](https://react-dnd.github.io/react-dnd/docs-tutorial.html) to `Adding the Drag and Drop Interaction` step
2. Add changes to `Board` and `Knight` components (I've used ES6 version)
3. Perform drag and drop on Knight

**Expected behavior**
Should look like this:
![dnd-original](https://user-images.githubusercontent.com/2122619/45006007-06c29b00-aff7-11e8-8bd0-e3b8a72a672c.gif)

**Screenshots**
Looks like that:
![dnd](https://user-images.githubusercontent.com/2122619/45006022-2063e280-aff7-11e8-8ae0-90a14d9b330a.gif)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome - 68.0.3440.106
 - react-dnd - 5.0.0
 - react-dnd-html5-backend - 5.0.1

Anyone else experienced similar issue?

**Comment by cman6397** [comment]
Yes I am experiencing this with a chess game I am making.  The issue does not seem to be happening in Firefox.  I really hope someone figures this out.  
![chess_drag_drop](https://user-images.githubusercontent.com/17400227/47833574-ccf0e500-dd58-11e8-8467-9e4495438daa.png)

**Comment by cman6397** [comment]
Hey guys, the work-around is to add a custom drag layer.  Check out the very end of the knight move tutorial.  just set the drag layer to whatever you want to drag.

**Comment by ssmlee04** [comment]
@cman6397 I am facing the same issue care to share where exactly in the docs are you referring to?

**Comment by cman6397** [comment]
https://react-dnd.github.io/react-dnd/docs/tutorial

Go to the final touches section.
